src/main/java/longah/commands/delete/DeleteGroupCommand.java

```diff
@@ -23,5 +23,6 @@ public DeleteGroupCommand(String commandString, String taskExpression) {
      *
      */
     public void execute(Group group) throws LongAhException {
-        GroupList.deleteGroup(this.taskExpression);}
+        GroupList.deleteGroup(this.taskExpression);
```

Comment:
Can add user feedback to this as well.

Reply:
This is handled within deleteGroup in GroupList itself